src/tests/Common/wasm-test-runner/WasmTestRunner.proj

```diff
@@ -1,6 +1,6 @@
 <!-- This project requires an explicit SDK version number because it is used on Helix,
       and global.json is not available. -->
-<Project Sdk="Microsoft.Build.NoTargets/1.0.53" DefaultTargets="WasmBuildApp">
+<Project Sdk="Microsoft.Build.NoTargets" DefaultTargets="WasmBuildApp">
```

Comment:
Err we do need a version here because this project is built on helix, without a global.json . Setting this to match what we have in `global.json`.

```suggestion
<Project Sdk="Microsoft.Build.NoTargets/3.5.0" DefaultTargets="WasmBuildApp">
  <!-- Set the version for NoTargets here explicitly because this is built on helix, without a global.json -->
```